### PR TITLE
Add 4.6 Insights Operator Release Notes

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to branch/enterprise-4.6 only.

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

**Comment by pneedle-rh** [comment]
@vikram-redhat and @sjstout can you please review this PR since merging is blocked for this branch? Thanks!

**Comment by pneedle-rh** [comment]
@radekvokal can you please review this PR from a PM perspective?

**Comment by pneedle-rh** [comment]
@tremes can you please review this PR from a SME perspective? Thanks!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1858,6 +1881,11 @@ The Ingress Operator was changed to consider unspecified values and default valu
 
 * Previously, there was no validation to prevent the Local Storage Operator from creating one device to multiple PVs. This release adds validation for this scenario so that trying to create a PV on a block device where one is already provisioned by the Local Storage Operator will fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1744385[*BZ#1744385*])
 
+*Insights Operator*
+
+* Previously, the Insights Operator could gather data for an unlimited number of certificate signing requests (CSRs) in a single report. This resulted in excessive data collection for clusters with many CSRs. Insights Operator now gathers data for a maximum of 5000 CSRs in a single report.
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:1886]: s/CSRs. Insights Operator/CSRs. The Insights Operator

**Comment by sjstout** [comment]
LGTM!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
...
+* JSON dumps for pods that have crash looping containers
+* The MachineSet resource configuration
+* The anonymized HostSubnet resource configuration
+* The MachineConfigPools configuration
+* The InstallPlans and their count
+* ServiceAccounts statistics from Openshift namespaces
+
+Additionally, with this release the Insights Operator collects information about all cluster nodes, while previous versions only collected information about unhealthy nodes.
+
+With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
```
Comment (radekvokal) [inline release_notes/ocp-4-6-release-notes.adoc:1180]: This should specifically point to cloud.redhat.com/openshift or talk about OpenShift Cluster Manager.

**Review by codyhoag — DISMISSED** [review]
One nit; everything else LGTM from a formatting/wording perspective :heavy_check_mark:

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
...
+
+[id="ocp-4-6-insights-operator-data-collection-enhancements”]
+==== Insights Operator data collection enhancements
+
+In {product-title} 4.6, the Insights Operator collects the following additional information:
+
+* Pod disruption budgets
+* The volume snapshot custom resource definition
+* The latest pod logs from pods that are not healthy
+* Running image data, including the number of containers using an image and the age of the corresponding pods
```
Comment (tremes) [inline release_notes/ocp-4-6-release-notes.adoc:1170]: I think this is only for Red Hat images right @natiiix ?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
 
 The Virtual Machine `Pending Changes` alert is now more informative. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1862801[*BZ#1862801*])
 
+[id="ocp-4-6-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-6-insights-operator-data-collection-enhancements”]
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:1162]: @pneedle-rh Not sure what is throwing off the build here. The only thing I noticed was the closing quotations in the ID don't match. That last one appears to be a special char version copied probably from a Google Doc. I would try replacing that and seeing if that fixes it (if you, too, don't know what's causing the issue).

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
...
+
+* Pod disruption budgets
+* The volume snapshot custom resource definition
+* The latest pod logs from pods that are not healthy
+* Running image data, including the number of containers using an image and the age of the corresponding pods
+* JSON dumps for pods that have crash looping containers
+* The MachineSet resource configuration
+* The anonymized HostSubnet resource configuration
+* The MachineConfigPools configuration
+* The InstallPlans and their count
```
Comment (tremes) [inline release_notes/ocp-4-6-release-notes.adoc:1175]: This is only for default and `openshift` namespaces.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc:1180]: @radekvokal thank you for your review. The `{cloud-redhat-com}` attribute renders as 'Red Hat OpenShift Cluster Manager' in the docs build. In the published document, the line will read as follows:

"With this additional information, Red Hat can provide improved remediation steps in Red Hat OpenShift Cluster Manager."

File: release_notes/ocp-4-6-release-notes.adoc (hunk 4 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: @codyhoag thank you for your review and for your guidance on this. I have just pushed an update after replacing that character and will monitor the progress of the new build.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 3 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: Thank you. I have now updated this line to reflect that.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: Thank you. I have now updated this line accordingly.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: Resolved.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 4 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: The subsequent build has succeeded. Thanks again!

**Review by codyhoag — APPROVED** [review]

File: release_notes/ocp-4-6-release-notes.adoc (hunk 3 above)
Comment (pneedle-rh) [reply release_notes/ocp-4-6-release-notes.adoc]: From [this PR](https://github.com/openshift/insights-operator/pull/133):

"Only information about images related to Red Hat will be collected in order to further lower the data size, but most importantly in order to maintain data privacy."

**Review by vikram-redhat — APPROVED** [review]

**Pull request merged**